templates/ts-bootstrap-cheerio-crawler/.eslintrc

```diff
@@ -1,4 +1,20 @@
 {
     "root": true,
-    "extends": "@apify/eslint-config-ts"
+    "env": {
+        "browser": true,
+        "es2020": true,
+        "node": true
+    },
+    "extends": [
+        "@apify/eslint-config-ts"
+    ],
+    "parserOptions": {
+        "project": "./tsconfig.json",
+        "ecmaVersion": 2020
+    },
+    "ignorePatterns": [
+        "node_modules",
+        "dist",
+        "**/*.d.ts"
+    ]
```

Comment:
do you really need those changes? you shouldn't need most of them i think. and if you need some, it might be better to modify the shared configs.

https://github.com/apify/apify-eslint-config/blob/master/index.js
https://github.com/apify/apify-eslint-config-ts/blob/master/index.js

looking at them, we already set some of those options.

Reply:
In this case, I copy pasted .eslintrc from another template (it is the same in all other templates)
Shortcuts:
- *a-config* = https://github.com/apify/apify-eslint-config/blob/master/index.js
- *a-config-ts* = https://github.com/apify/apify-eslint-config-ts/blob/master/index.js

As I see it, necessary would be only:
- env: es2020: true
  - or it could be added to a-config
- extends: @apify/eslint-config-ts
- parserOptions: ecmaVersion: 2020 
  - ?? I am not sure about this one - it is in a-config, but iparserOptions are under "overrides" in a-config-ts without ecmaVersion: 2020 so probably necessary here, but could be added to a-config-ts
- ignorePatterns

Do you think I should change it?
For the record, I joined Apify two weeks ago and don't want to make any significant changes.